numpy/random/_pcg64.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import Dict, Literal, Union
```

Comment:
```suggestion
import sys
from typing import Dict, Union

if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```
`Literal` was only added to `typing` in python 3.8; for older versions it's still available in `typing_extensions` however.
Note that this comment also applies to all other files.